code/string-calculator/view/src/main/java/ru/unn/agile/calc/view/StringCalcView.java

```diff
@@ -0,0 +1,44 @@
+package ru.unn.agile.calc.view;
+
+import javafx.css.PseudoClass;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.TextField;
+import javafx.scene.control.Tooltip;
+import ru.unn.agile.calc.viewmodel.StringCalcViewModel;
+
+public class StringCalcView {
+
+    private final PseudoClass errorClass = PseudoClass.getPseudoClass("error");
+    private final String tooltip = "Type your expression here";
```

Comment:
Please change to the clear description of valid input

Reply:
fixed